source/20_spec/config_spec_en.rst

```diff
@@ -0,0 +1,139 @@
+DI-engine Config Specification
+===============================
+
+The config submitted by the developer needs to comply with the following specifications to ensure the ease of use, readability, and extensibility of the config.
+
+The config of DI-engine includes two parts: main_config and create_config.
+
+example link
+-------------
+
+Example of a common algorithm (DQN):
+
+https://github.com/opendilab/DI-engine/blob/main/dizoo/atari/config/serial/pong/pong_dqn_config.py
+
+Example of an algorithm (SQIL) containing a model or data:
+
+https://github.com/opendilab/DI-engine/blob/main/dizoo/atari/config/serial/pong/pong_sqil_config.py
+
+normative content
+------------------
+
+Grammar specification
+~~~~~~~~~~~~~~~~~~~~~~
+
+-  config needs to satisfy flake8 python syntax check and yapf format.
+
+Naming conventions
+~~~~~~~~~~~~~~~~~~~
+
+-  config.py file name, main_config and create_config related variable names
+
+   -  unified with <environment name>\_<algorithm name>\_config.py. The name of the file and related variable names in the file do not need to add the default field. For example, the filename hopper_onppo_default_config.py should be changed to hopper_onppo_config.py.
+
+   -  similar For the ICM algorithm, the general algorithm is the module proposed in the paper combined with a baseline algorithm, and the corresponding config name is in accordance with <environment name>\_<module name>\_<baseline algorithm name>\_config.py Name it like cartpole_icm_offppo_config.py
+
+   -  If the algorithm has different versions of on-policy and off-policy, unify the file name of config.py and the relevant variable name in the file, and use onppo/offppo to distinguish the algorithm of on-policy and off-policy version. For example, for the config of the PPO algorithm, hopper_ppo_config.py should be changed to hopper_onppo_config.py.
+
+-  exp_name field
+
+   -  exp_name field must be added in main_config
+
+   -  The naming convention is environment+algorithm+seed, eg:\ ``qbert_sqil_seed0``
+
+-  file pathname
+
+   -  See the sql example and comment accordingly. If multiple models need to be loaded, the model path (model_path) variables are named as follows: prefix1_model_path, prefix2_model_path, ..., The data path (data_path) variable is named similarly.
```

Comment:
Soft Q Learning might be better since sql is more renowned as Structured Query Language.